README.md

```diff
@@ -1,6 +1,6 @@
 # RKM - Rust k-means #
 
-[![docs](https://docs.rs/rkm/badge.svg)](https://docs.rs/rkm/latest/rkm/) [![crates.io](https://img.shields.io/crates/v/rkm.svg)](https://crates.io/crates/rkm)
+[![Build Status](https://travis-ci.org/emmanuelantony2000/rkm.svg?branch=master)](https://travis-ci.org/emmanuelantony2000/rkm) [![docs](https://docs.rs/rkm/badge.svg)](https://docs.rs/rkm/latest/rkm/) [![crates.io](https://img.shields.io/crates/v/rkm.svg)](https://crates.io/crates/rkm)
```

Comment:
Please replace this with https://travis-ci.org/genbattle/rkm.svg?branch=master. I've set up Travis for this repo, so once the configuration merges the build status will be displayed.

Reply:
I'll do that too.. and i used rustfmt for formatting...